public/services/utils/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DataSourceOption } from 'src/plugins/data_source_management/public/components/data_source_menu/types';
+
+export const dataSourceInfo: { activeDataSource: DataSourceOption } = {
+  activeDataSource: {
+    id: '',
```

Comment:
just curious, does this mean activeDataSource by default will be local cluster? since it's id is ' ' string

Reply:
+1

Reply:
Yes, by default it is so. I am working on this piece to figure out the right way to set the default source from the plugin.